### PR TITLE
Warn sysadmins about sensitive information stored on server.

### PR DESCRIPTION
**Pull request by mtbc**
See https://trello.com/c/0CixF1ge/25-encrypt-uuids-in-db. Staged at https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ws/src/omero/_build/html/sysadmins/server-backup-and-restore.html#backing-up-your-postgresql-database and https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ws/src/omero/_build/html/sysadmins/server-security.html#stored-data.

### CODE REVIEW

**Comment by mtbc** [comment]
Mostly just states the obvious: after all, the database also stores everything from private groups, everybody's comment on everybody's image, etc. -- clearly the server needs to store this stuff *somewhere* -- but never hurts to be more explicit for those who just hadn't really thought about it.

**Comment by joshmoore** [comment]
Certainly no objections to the addition of the text. Longer term I think we will have to improve the visibility (and the readability in general)

File: omero/sysadmins/cli/light-admins.rst [hunk 1]
```diff
@@ -65,6 +65,8 @@ removes a restriction so that they may exercise `SomePrivilege`.
   You may not clear a restriction from an administrator if you have that
   same restriction applying to yourself.
 
+.. _ReadSession-warning:
```
Comment (pwalczysko) [inline omero/sysadmins/cli/light-admins.rst:68]: not sure why is it here ? This is not visible in the built version. The other warnings are called just "warning", even here in the doc ?

Comment (pwalczysko) [reply omero/sysadmins/cli/light-admins.rst:68]: aha, got it, you need this anchor to be able to call the warning from the https://github.com/openmicroscopy/ome-documentation/pull/1930/files#diff-0ff5c8ce87cff45a62568c35b1f915d8

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**